frontend/public/components/dashboard/dashboards-page/cluster-dashboard/status-card.tsx

```diff
@@ -139,8 +142,8 @@ const ClusterAlerts = connectToFlags(FLAGS.OPENSHIFT)(
       return (
         <AlertsBody
           isLoading={
-            flagPending(isOpenshift) ||
-            (isOpenshift ? !(alertsResponse && cvLoaded) : !alertsResponse)
+            flagPending(hasCVResource) ||
+            !alertsResponse || (hasCVResource && !cvLoaded && !cvError)
```

Comment:
Do we need to be checking for alert response errors, too?

Reply:
Nevermind, I see `AlertsBody` handles that

Reply:
if alerts fetching fails, then we set the `error` prop one line below which shows `Alerts could not be loaded.`
We could do the same for `cvError` but right now Im treating fetching that resource as optional - ie if the fetching cv fails, we can still show alerts.